recipes/dlib/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "dlib" %}
+{% set version = "19.0" %}
+{% set blas_variant = "openblas" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/davisking/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: a6463524cdb1d275ab408f182b0d47af143179fde7f1dd97074dc25f2f9a2676
+
+  patches:
+    - osx_jpeg.patch                # [osx]
+    - lapack_sgetrf.patch           # [unix]
+    - win_boost.patch               # [win]
+    - win_png.patch                 # [win]
+    - add_global_cxx_release.patch
+
+build:
+  number: 0
+  features:  # [not win]
+    - blas_{{ blas_variant }}  # [not win]
+
+requirements:
+  build:
+   - python
+   - cmake
+   - toolchain
+   - openblas              # [not win]
+   - numpy >=1.10
+   - boost 1.61.*
+   - jpeg 9*               # [not win]
+   - libpng >=1.6.21,<1.7
+   - sqlite 3.13.*         # [not win]
+
+  run:
+   - python
+   - openblas              # [not win]
+   - numpy >=1.10
+   - boost 1.61.*
+   - jpeg 9*               # [not win]
+   - libpng >=1.6.21,<1.7
+   - sqlite 3.13.*         # [not win]
+
+test:
+  files:
+    - test_data/face.jpg
+    - test_data/face.png
+    - test_data/images.xml
+
+  requires:
+    - tqdm
+    - pillow
+
+  imports:
+    - dlib
+
+about:
+  home: http://dlib.net/
+  license: Boost Software License
+  summary: Header only machine learning toolkit
```

Comment:
Seems like it some `cpp` files at least for binding to Python if nothing else. Maybe we can use the repo description line or similar.